File: sdkwork-ai-sdk-csharp/Models/RequestCounts.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class RequestCounts
    {
        public int? Total { get; set; }
        public int? Completed { get; set; }
        public int? Failed { get; set; }
    }
}

File: sdkwork-ai-sdk-java/src/main/java/com/sdkwork/ai/model/RequestCounts.java
package com.sdkwork.ai.model;

public class RequestCounts {
    private Integer total;
    private Integer completed;
    private Integer failed;

    public Integer getTotal() {
        return this.total;
    }
    
    public void setTotal(Integer total) {
        this.total = total;
    }

    public Integer getCompleted() {
        return this.completed;
    }
    
    public void setCompleted(Integer completed) {
        this.completed = completed;
    }

    public Integer getFailed() {
        return this.failed;
    }
    
    public void setFailed(Integer failed) {
        this.failed = failed;
    }
}
